boards/common/particle-mesh/Makefile.include

```diff
@@ -1,3 +1,9 @@
+# This board's nRF radio comes with a uFL antenna plug and a built-in antenna.
```

Comment:
Maybe some words regarding this antenna selection could be added in the boards documentation.